### ISSUE TITLE
NTR: 3-hydroxykynurenine high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8553]
New term request 3-hydroxykynurenine high in blood (elthat_171215053014):

**Comment by pnrobinson** [comment]
@MickeySegal
if this is referring to Hydroxykynureninuria (OMIM:236800), then the increase in 3-hydroxykynurenine was found in two healthy Somali siblings but in urine rather than in the ciurculation. This seems very unlikely as a clinical finding, I do not think we really need a term for it?

See: https://pubmed.ncbi.nlm.nih.gov/17334708/

**Comment by MickeySegal** [comment]
https://www.omim.org/clinicalSynopsis/617661 "Increased serum levels of 3-hydroxykynurenine (3HK)"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000776
- label: Elevated circulating 3-hydroxykynurenine concentration
- parents: [Abnormal circulating non-proteinogenic amino acid concentration (HP:0033109)](https://hpo.jax.org/app/browse/term/HP:0033109)
- pmid: PMID:28792876
- def: The concentration of 3-hydroxykynurenine in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8553
- synonyms:
